Update InstallFullDB.config

Zeroradiance opened the pull request:
Updated to match changes to entrypoint to fix MYSQL / Mariadb

Byloth commented:
Did you experience any problem with this project?
Want to contribute to fix / improve the project itself?
Want to add support for AHBots and PlayerBots? <sup>1</sup>

[Let's speak](https://discord.com/users/538459351111827457) about it! 😉

---

1. Did you notice? There's already a draft PR AHBots & PlayerBots: https://github.com/Byloth/cmangos-docker/pull/20

Zeroradiance commented:
Thank you for discussing this with me over Discord to clear up the situation 😄, I will leave this here as it may help others who run into the issue.

MariaDB 11.5+ completely removed the old mysql and mysqldump commands, replacing them with mariadb and mariadb-dump

In InstallFullDB.config:
Changed the paths from /usr/bin/mysql and /usr/bin/mysqldump to /usr/bin/mariadb and /usr/bin/mariadb-dump

In entrypoint.sh:
Updated the mysql_execute() and mysql_dump() functions to call mariadb and mariadb-dump instead of the old commands.

After making these changes, everything works perfectly database initialization completes successfully and services connect without issues.
